src/pretalx/settings.py

```diff
@@ -11,28 +10,98 @@
 
 
 config = configparser.RawConfigParser()
+config.read_dict({
+    'filesystem': {
+        'base': os.path.dirname(os.path.dirname(__file__)),
+    },  # defaults depend on the data dir and need to be set once the data dir is fixed
+    'site': {
+        'debug': 'runserver' in sys.argv,
+        'url': 'http://localhost',
+        'cookie_domain': '',
+    },  # the https setting is determined by url if not explicitly set
+    'database': {
+        'backend': 'sqlite3',
+        # 'name': '',
+        'user': '',
+        'password': '',
+        'host': '',
+        'port': '',
+    },
+    'mail': {
+        'from': 'admin@localhost',
+        'host': 'localhost',
+        'port': '25',
+        'user': '',
+        'password': '',
+        'tls': 'False',
+        'ssl': 'True',
+    },
+    'cache': {
+
+    }
+})
+
+
 if 'PRETALX_CONFIG_FILE' in os.environ:
     config.read_file(open(os.environ.get('PRETALX_CONFIG_FILE'), encoding='utf-8'))
 else:
-    config.read(['/etc/pretalx/pretalx.cfg', os.path.expanduser('~/.pretalx.cfg'), 'pretalx.cfg'],
-encoding='utf-8')
+    config.read([
+        '/etc/pretalx/pretalx.cfg',
+        os.path.expanduser('~/.pretalx.cfg'),
+        'pretalx.cfg',
+    ], encoding='utf-8')
+
+env_config = {
+    'filesystem': {
+        'data': os.getenv('PRETALX_DATA_DIR'),
```

Comment:
Most of these env variables are introduced in this MR, right? Is there a reason to not make them match the config file options? e.g. `[filesystem] data` => `PRETALX_FILESYSTEM_DATA`.

Reply:
Nope, most of these options have been around already, which is the reason for the somewhat inconsistent naming.